.evergreen/setup-env.sh

```diff
@@ -12,11 +10,12 @@ evergreen_root="$(pwd)"
 : "${WINDOWS_32BIT:=}"
 : "${OS:=unspecified}"
 
-[ -d "${MONGOCRYPT_INSTALL_PREFIX:=${evergreen_root}/install/libmongocrypt}" ] || mkdir -p "${MONGOCRYPT_INSTALL_PREFIX}"
+evergreen_root="$(dirname "$LIBMONGOCRYPT_DIR")"
```

Comment:
`evergreen_root` mean something different from `EVG_DIR` in this context. Here it's looking for the EVG workspace directory, which is coincidentally the parent directory where we put `libmongocrypt`, since the source checkout clones into `<workdir>/libmongocrypt`. I'd very much like the build to _never_ reach outside of the source repo, but that will require several more changes that aren't necessary for this to work as-is.

Reply:
Oops. I misread.